src/main/java/frc/robot/commands/SetTrapLineupPosition.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.commands;
+
+import java.util.Optional;
+
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.drive.DriveSubsystem;
+import frc.robot.subsystems.intakeLimelight.IntakeLimelight;
+
+public class SetTrapLineupPosition extends Command{
+  private DriveSubsystem drive;
+  private IntakeLimelight tagLimelight;
+  private boolean targetLocked = false;
+
+  //Set the final position in the drive subsystem to the location of the tag the limelight sees
```

Comment:
/** */ style comments for classes, variables, and functions. // style comments for commenting code. Also I think this comment is applying to the class and not the constructor